CONTRIBUTING.md

```diff
@@ -3,8 +3,11 @@
 🙏 We would ❤️ for you to contribute to Falso and help make it even better than it is today!
 
 ## Developing
+
 - Run `npm i`
-- Create new falso by running `yarn f name`
+- Create new falso by running `yarn new --name name --json`
+
+  - Use `--json` (default `false`) to indicate that the falso will load it's data from a JSON file
```

Comment:
Just a question here: Won't all falso's in the future will have to serve data from a static JSON file?

Reply:
No, many of the functions will generate the data programmatically